### PR TITLE
0.5.0 Docs 

### PR DESCRIPTION
**Pull request by Lagoja**
## Summary
Includes Docs for:

1. Version selection
2. `includes` 
3. Updated Devbox Cloud
4. CLI Changes

## How was it tested?

Localhost

### CODE REVIEW

File: docs/app/docs/configuration.md [hunk 1]
```diff
@@ -130,14 +147,27 @@ To run multiple commands in a single script, you can pass them as an array:
...
+{
+    "includes": [
+        "plugin:php-config"
+    ]
+}
+```
 
+### Nixpkgs
+
+The Nixpkg object is used to optionally configure which version of the Nixpkgs repository you want Devbox to use as the default for installing packages. It currently takes a single field, `commit`, which takes a commit hash for the specific revision of Nixpkgs you want to use.
```
Comment (LucilleH) [inline docs/app/docs/configuration.md:166]: Aren't we getting rid of this now that we have a lock file?

File: docs/app/docs/devbox_cloud/beta_faq.md [hunk 2]
```diff
@@ -5,29 +5,31 @@ sidebar_position: 4
...
 ### Do I have to pay to use Devbox Cloud during the Open Beta?
 
-Devbox Cloud is free to use during the Open Beta period, subject to the restrictions listed below. We expect to continue offering a free tier for personal use after the Open Beta period, but we will offer Paid Plans that provide more resources, concurrency, and persistence. 
+Devbox Cloud is free to use during the Open Beta period, subject to the restrictions listed below. We expect to continue offering a free tier for personal use after the Open Beta period, but we will offer Paid Plans that provide more resources, projects, and persistence.
 
 ### What are the resource limits for Devbox Cloud VMs
 
-* **CPU**: 1 Core, shared
-* **RAM**: 1 GB
+* **CPU**: 4 Cores
```
Comment (LucilleH) [inline docs/app/docs/devbox_cloud/beta_faq.md:31]: still shared

File: docs/app/docs/devbox_cloud/browser_getting_started.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ If you are trying to open a private repository in Devbox Cloud, you will first h
 :::
 
 1. Navigate to the Github Repo that you want to preview in Devbox Cloud
-2. Copy the URL, and prepend it with `https://devbox.sh/`. For example, to open the Devbox repo, you would enter [https://devbox.sh/github.com/jetpack-io/devbox](http://devbox.sh/github.com/jetpack-io/devbox)
+2. Copy the URL, and prepend it with `https://devbox.sh/`. For example, to open the Devbox repo, you would enter [https://devbox.sh/github.com/jetpack-io/devbox](http://devbox.sh/github.com/jetpack-io/devbox). You can also enter the URL on [devbox.sh](https://devbox.sh) and click the `Open` button
```
Comment (LucilleH) [inline docs/app/docs/devbox_cloud/browser_getting_started.md:17]: Click the `Go` button (or should we change the button to `Open`?)

**Review by LucilleH — COMMENTED** [review]
I don't see where we ask users to commit the lock file?

File: docs/app/docs/devbox_cloud/browser_getting_started.md (hunk 3 above)
Comment (Lagoja) [reply docs/app/docs/devbox_cloud/browser_getting_started.md:17]: I'll edit it to "Go"

**Comment by Lagoja** [comment]
> I don't see where we ask users to commit the lock file?

See latest commit, let me know if you think we should mention it in more places.

**Review by LucilleH — APPROVED** [review]

File: docs/app/docs/devbox_cloud/browser_getting_started.md (hunk 3 above)
Comment (LucilleH) [reply docs/app/docs/devbox_cloud/browser_getting_started.md:17]: Not changed @Lagoja ?

**Pull request merged**